editor/src/components/canvas/remix/remix-rendering.spec.browser2.tsx

```diff
@@ -49,3 +170,94 @@ describe('Remix content with feature switch off', () => {
     ).rejects.toThrow()
   })
 })
+
+describe('Remix navigation', () => {
+  setFeatureForBrowserTestsUseInDescribeBlockOnly('Remix support', true)
+  it('Can navigate to a different route', async () => {
+    const project = createModifiedProject({
+      [StoryboardFilePath]: `import * as React from 'react'
+        import { RemixScene, Storyboard } from 'utopia-api'
+        
+        export var storyboard = (
+          <Storyboard>
+            <RemixScene
+              style={{
+                width: 700,
+                height: 759,
+                position: 'absolute',
+                left: 212,
+                top: 128,
+              }}
+              data-label='Playground'
+            />
+          </Storyboard>
+        )
+        `,
+      ['/src/root.js']: `import React from 'react'
+        import { Outlet } from '@remix-run/react'
+        
+        export default function Root() {
+          return (
+            <div>
+              This is root!
+              <Outlet />
+            </div>
+          )
+        }
+        `,
+      ['/src/routes/_index.js']: `import React from 'react'
+        import { Link } from '@remix-run/react'
+  
+        export default function Index() {
+          return <Link to='/about' data-testid='remix-link'>${DefaultRouteTextContent}</Link>
+        }
+        `,
+      ['/src/routes/about.js']: `import React from 'react'
+  
+        export default function About() {
+          return <h1>About</h1>
+        }
+        `,
+    })
+
+    const renderResult = await renderTestEditorWithModel(project, 'await-first-dom-report')
+    await renderResult.dispatch([switchEditorMode(EditorModes.liveMode())], true)
+
+    const targetElement = renderResult.renderedDOM.getByTestId('remix-link')
+    const targetElementBounds = targetElement.getBoundingClientRect()
+
+    const clickPoint = windowPoint({ x: targetElementBounds.x + 5, y: targetElementBounds.y + 5 })
+
+    await mouseClickAtPoint(targetElement, clickPoint)
+
+    await expectRemixSceneToBeRendered(renderResult, 'About')
+  })
+})
+
+async function dragElement(
+  canvasControlsLayer: HTMLElement,
+  startPoint: WindowPoint,
+  dragDelta: WindowPoint,
+  modifiers: Modifiers,
+  midDragCallback: () => Promise<void>,
+): Promise<void> {
+  await mouseDownAtPoint(canvasControlsLayer, startPoint, { modifiers: cmdModifier })
+  await mouseDragFromPointWithDelta(canvasControlsLayer, startPoint, dragDelta, {
+    modifiers,
+    midDragCallback,
+  })
+}
+
+async function expectRemixSceneToBeRendered(
+  editor: EditorRenderResult,
+  textContent: string = DefaultRouteTextContent,
+) {
+  const [remixScene] = await editor.renderedDOM.findAllByTestId(REMIX_SCENE_TESTID)
+  expect(remixScene).toBeDefined()
+
+  const [rootText] = await editor.renderedDOM.findAllByText('This is root!')
```

Comment:
Minor point, it seems like the `This is root!` should be a constant in this file